docs/content/stable/yugabyte-platform/security/create-kms-config/azure-kms.md

```diff
@@ -54,7 +55,7 @@ If you are planning to use an existing cryptographic key with the same name, it
 - Permitted operations should have at least WRAP_KEY and UNWRAP_KEY.
 - The key rotation policy should not be defined in order to avoid automatic rotation.
 
-You can create a KMS configuration that uses Azure KMS, as follows:
+You can create a KMS configuration that uses Azure Key Vault, as follows:
 
 1. Use the YugabyteDB Anywhere UI to navigate to **Configs > Security > Encryption At Rest** to access the list of existing configurations.
 
```

Comment:
Line 67 - **KMS Provider** — Select **Azure KMS** . This needs to be changed. Present in stable version too